support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -54,13 +54,12 @@ DomainCreator:
         ExciseWithBoundaryCondition:
           DemandOutgoingCharSpeeds:
       UseLogarithmicMap: true
-    # For the envelope, setting radial distribution to logarithmic produces
-    # lower constraints than other radial distrubtions.
+    # Radial distribution is linear to resolve gravitational waves
     Envelope:
       Radius: 100.0
       RadialDistribution: Logarithmic
-    # For the outer shell, setting radial distribution to linear produces lower
-    # constraints than other radial distrubtions.
+    # Radial distribution is logarithmic to transition from grid point spacing
+    # around the excisions to the grid point spacing in the outer shell
```

Comment:
You swapped these two comments. Put them right above their corresponding `RadialDistribution` line, so it's clear what line they comment on.